### PR TITLE
Added tests for raw per-cpu VPA data, maximum memory value via debugfs

### PR DESCRIPTION
**Pull request by shirishaganta1**
Tests whether files are generated for all cpu's and
maximum memory values are same or not

Signed-off-by: Shirisha Ganta <SHIRISHA.Ganta1@ibm.com>

### CODE REVIEW

**Comment by shirishaganta1** [comment]
output
\# avocado run vpa_MaxMem_check.py 
JOB ID     : 37cb26d3457c857ec7516a46fbbed0ed7260075b
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2021-09-22T17.51-37cb26d/job.log
 (1/1) vpa_MaxMem_check.py:Cpu_VpaData.test: PASS (0.50 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado-fvt-wrapper/results/job-2021-09-22T17.51-37cb26d/results.html
JOB TIME   : 11.07 s

File: ras/vpa_MaxMem_check.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    @skipIf(IS_POWER_NV, "This test is supported on PowerVM environment")
+    def setUp(self):
+        detected_distro = distro.detect()
+        if detected_distro.name not in ['rhel', 'SuSE']:
+            self.cancel("Test case is supported only on RHEL and SLES")
+        
+    def test(self):
+
+        self.log.info("===Checking for cpu VPA data and MaxMem==")
+        process.system('ls /sys/kernel/debug/powerpc/vpa')
```
Comment (PraveenPenguin) [inline ras/vpa_MaxMem_check.py:39]: please use python way to access a file or i say please use `genio` avocado library for same

File: ras/vpa_MaxMem_check.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        cpu_count = process.system_output("ls /sys/kernel/debug/powerpc/vpa | wc -l", shell=True)
```
Comment (PraveenPenguin) [inline ras/vpa_MaxMem_check.py:40]: here as well please python way

File: ras/vpa_MaxMem_check.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        process.system('cat /proc/powerpc/lparcfg')
+        maxmem = process.system_output("cat /proc/powerpc/lparcfg | "
+                                       "grep MaxMem | "
+                                       "cut -d'=' -f2", shell=True).strip()
+        total_cpus = process.system_output("lscpu | awk '{if(NR==3) print $2}'", shell=True)
```
Comment (PraveenPenguin) [inline ras/vpa_MaxMem_check.py:45]: please use the python `re` module to do the operation as embedding  bash command in python ,not a good practice as it use subprocess module it kind of more over head for CPU

File: ras/vpa_MaxMem_check.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        lmaxmem = process.system_output("lparstat -i | "
+                                        "grep \"Maximum Memory\" | "
```
Comment (PraveenPenguin) [inline ras/vpa_MaxMem_check.py:47]: here also same

**Review by PraveenPenguin — CHANGES_REQUESTED** [review]

File: ras/vpa_MaxMem_check.py (hunk 4 above)
Comment (shirishaganta1) [reply ras/vpa_MaxMem_check.py:47]: After addressing all the above comments, changed the script. Please check it once Praveen

**Comment by shirishaganta1** [comment]
\# avocado run vpa_MaxMem_check.py 
JOB ID     : 39aae89354749399b41ddd1428a4282cb52ba8cf
JOB LOG    : /home/siri/avocado-fvt-wrapper/results/job-2021-10-05T01.33-39aae89/job.log
 (1/1) vpa_MaxMem_check.py:Cpu_VpaData.test: PASS (0.50 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/siri/avocado-fvt-wrapper/results/job-2021-10-05T01.33-39aae89/results.html
JOB TIME   : 8.93 s

**Review by PraveenPenguin — APPROVED** [review]
LGTM

**Pull request merged**